src/phoebus_utils/adiabats.hpp

```diff
@@ -0,0 +1,132 @@
+// © 2021. Triad National Security, LLC. All rights reserved.  This
+// program was produced under U.S. Government contract
+// 89233218CNA000001 for Los Alamos National Laboratory (LANL), which
+// is operated by Triad National Security, LLC for the U.S.
+// Department of Energy/National Nuclear Security Administration. All
+// rights in the program are reserved by Triad National Security, LLC,
+// and the U.S. Department of Energy/National Nuclear Security
+// Administration. The Government is granted for itself and others
+// acting on its behalf a nonexclusive, paid-up, irrevocable worldwide
+// license in this material to reproduce, prepare derivative works,
+// distribute copies to the public, perform publicly and display
+// publicly, and to permit others to do so.
+
+#ifndef PHOEBUS_UTILS_ADIABATS_HPP_
+#define PHOEBUS_UTILS_ADIABATS_HPP_
+
+#include <limits>
+
+#include <globals.hpp>
+#include <kokkos_abstraction.hpp>
+#include <utils/error_checking.hpp>
+
+#include <spiner/databox.hpp>
+
+#include "microphysics/eos_phoebus/eos_phoebus.hpp"
+#include "phoebus_utils/root_find.hpp"
+
+// a namespace?
+using Microphysics::EOS::EOS;
```

Comment:
As a general rule, it's not great to have a `using` statement outside a namespace in a header file. Maybe hide all of this behind an `Adiabats` namespace or something similar?

Reply:
done